### DISCUSSION TITLE
Proposed tech stack

### DISCUSSION THREAD

**Discussion by kleenkanteen** [chingu-voyages/v50-tier3-team-24#15] [Ideas]
Here is mine:

frontend: nuxt + vue 

why vue over react?
cool vid about nuxt (nextjs but for vue): https://www.youtube.com/shorts/T-zau3dGjGE 
next vs. nuxt https://www.reddit.com/r/nextjs/comments/1aerwfx/comment/kkaqgrx/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button
also an explanation of vue's syntax and why its a lot cleaner than react: https://vuejs.org/guide/essentials/template-syntax.html

backend: nuxt api routes hosted 
on vercel + supabase 
(spins up a serverless postgres db for you, has a really nice dashboard and features)

more about supabase: https://www.youtube.com/watch?v=zBZgdTb-dns

**Comment by jkyan1988** [comment]
I'm okay with anything similar to React, as my knowledge of next.js is pretty new. If its just syntax that's different then I can adapt to whatever close to it. Backend, I don't have a preference, I'm probably more comfortable with frontend as of now since I'm getting back into coding.

**Comment by davideastmond** [comment]
For the front-end framework I strongly prefer React (NextJs ideally, but not necessary). Tailwind and MaterialUi are fine for me. 
For the back-end, MongoDb is my goto, but I'm open to another database solution - it depends if we decide to go with a relational or non-relational solution.

**Comment by Tandid** [comment]
Frontend - Ideally **NextJS** and **React**. However, I'm open to learning **Vue** and **Nuxt** even though I have no familiarity with it. For styling, I like to use **Tailwind**, but I'm also okay with **Material UI**.

Backend - I usually use **MongoDB** for databases. Non-relational databases are easier to utilize in my opinion, especially for complicated projects. I'm also open to learning how to utilize **Firebase** or **Appwrite** for storage.

**Comment by mnichols08** [comment]
Don't forget the project itself is not the be all and end all, that it has to be perfect so you can put it on your portfolio. A Voyage is about collaboration and communication. It's about understanding that YOU are accountable for the work that's assigned to you. It's about learning to understand that sometimes you have to ask for help, and sometimes you have to offer your help. This is a quote paraphrased from Jim. 

While everybody wants to complete a wonderful project during the Voyage remember that there is a safety net that allows us to experiment without risk. This is not the real world but rather a simulation.

What if you were hired to work a team because you impressed the hiring manager and they believed in you to quickly pick up this other framework?

With that said, this is a short six week project and there might be a reality that completing this project within six weeks while learning a new framework may or may not be feasible.

I think that it should ultimately still come down to a vote though

**Reply by Tandid** [reply]
This is a great point! I think I'm more convinced in regards to trying out a new tech stack now HAHA, so my vote will probably go towards Vue and Nuxt
